src/callback/file_callbacks.cpp

```diff
@@ -59,7 +59,7 @@ namespace Callbacks{
 
                 //We get the Godot File object from the handle
                 GodotFileHandle* handle {reinterpret_cast<GodotFileHandle*>(current_request->handle)};
-                godot::Ref<godot::File> file {handle->file};
+                godot::Ref<godot::FileAccess> file {handle->file};
```

Comment:
@piiertho `godot::File` got removed and instead, I am now using `godot::FileAccess`